retroshare-android-service/src/service.cpp

```diff
@@ -39,11 +39,19 @@ int main(int argc, char *argv[])
 #endif
 
 	QCoreApplication a(argc, argv);
+
+	RsInit::InitRsConfig();
```

Comment:
is there a special reason to call RsInit::InitRsConfig()?
IMHO it should not be necessary, because RsControlModule already calls it.